gui/MainWindow.cxx

```diff
@@ -838,6 +874,52 @@ void MainWindow::loadTracks(QString const& path)
 }
 
 //-----------------------------------------------------------------------------
+void MainWindow::loadDepthmaps(QString const& path)
+{
+  QTE_D();
+
+  QFile vtListiFile(path);
+  auto const& vtiListFileBasePath =  QFileInfo(path).absoluteDir().absolutePath();
+
+  if (vtListiFile.open(QIODevice::ReadOnly | QIODevice::Text))
+  {
+    QTextStream vtiStream(&vtListiFile);
+    int frameNum;
+    QString vtiFileName;
+
+    while (!vtiStream.atEnd())
+    {
+      vtiStream >> frameNum >> vtiFileName;
+
+      if(!vtiFileName.isEmpty())
+      {
+        vtiFileName = vtiListFileBasePath + QDir::separator() + vtiFileName;
+
+        d->cameras[frameNum].vtiPath = vtiFileName;
```

Comment:
Potential buffer overflow: need to check that `frameNum` is within the bounds of the list.